components/Navbar.jsx

```diff
@@ -7,24 +7,17 @@ export const Navbar = () => {
       <ul>
         <li>
           <Link href="/">
-            <a>
-              <img src="/Icons/MainCueCard-Black.svg" alt="Main Icon" />
-            </a>
+            <a> Home </a>
```

Comment:
I'd suggest removing this whitespace. I should not be visible in the DOM anyway.
```suggestion
            <a>Home</a>
```